src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -308,8 +307,33 @@ function MultiCombobox<Item>({
                         {itemToString(selectedItem)}
                       </Tag>
                     ))}
-                  </Flex>
-                )}
+                    <Box
+                      as="li"
+                      maxWidth="100%"
+                      flexGrow={1}
+                      minWidth="60px"
+                      position={isOpen && searchable ? 'relative' : 'initial'}
+                    >
+                      {isOpen && searchable && (
+                        <InputElement
+                          as="span"
+                          px={2}
+                          py={0}
+                          standalone={hideLabel}
+                          visibility="hidden"
+                          whiteSpace="pre"
+                        >
+                          {inputValue}
+                        </InputElement>
+                      )}
```

Comment:
This is the hack that will force the input to move to the next line when the content doesn't fit in the first line.
I used a hidden span that has the same content as the input in order to track the width of the input.